pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -575,7 +575,19 @@ public BookieAffinityGroupData getBookieAffinityGroup(@PathParam("property") Str
         validateNamespaceName(property, cluster, namespace);
         return internalGetBookieAffinityGroup();
     }
-    
+
+    @DELETE
+    @Path("/{property}/{cluster}/{namespace}/persistence/bookieAffinity")
+    @ApiOperation(hidden = true, value = "Delete the bookie-affinity-group from namespace-local policy.")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
+            @ApiResponse(code = 404, message = "Namespace does not exist"),
+            @ApiResponse(code = 409, message = "Concurrent modification") })
```

Comment:
@rdhabalia since you are adding new endpoints, do you mind making sure all the return codes are properly documented? since there are multiple efforts on improving the documentation around this area.

Reply:
yes, it seems correct to me.